spec/sepa_reason_codes/code_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SepaReasonCodes::Code do
+  describe '.==' do
```

Comment:
Style: instance methods usually are written with a hash, whereas class methods are a period. 

The method shorthand here should be `#==`

Reply:
changed :+1: